src/Command/CounterCacheCommand.php

```diff
@@ -73,7 +73,7 @@ public function execute(Arguments $args, ConsoleIo $io): int
         }
 
         /** @phpstan-ignore-next-line */
-        $table->updateCounterCache(...$methodArgs);
+        $table->behaviors()->get('CounterCache')->updateCounterCache(...$methodArgs);
```

Comment:
Is there a reason not to put the getter on table?

Reply:
None besides it wasn't really needed due to the method proxying. We could now add for e.g. `Table::getBehavior()` to shorten the call chain.

Reply:
isnt there a getBehavior() on the Table?
So

    $table->getBehavior('CounterCache')->updateCounterCache()
    
then?
Or do we need another method?

Reply:
Oh, there already is, my bad.

I'll update.